README.md

```diff
@@ -1,72 +1,298 @@
-# puppetlabs-common_events
-
-The Common Integration Events module collates different PuppetEnterprise API data sources into a common reporting Gem.
-
-PROTOTYPE!
+# puppetlabs-pe_event_forwarding
 
 #### Table of Contents
 
 
 1. [Intro](#intro)
 2. [Usage](#usage)
-3. [Pre-reqs](#pre-reqs)
-4. [Setup](#setup)
+3. [How it works](#how-it-works)
+4. [Writing an Event Forwarding Processor](#writing-an-event-forwarding-processor)
+5. [Installing A Processor](#installing-a-processor)
+6. [Token vs Username Password Authentication](#token-vs-username-password-authentication)
+7. [Resources Placed On The Machine](#resources-placed-on-the-machine)
+8. [Logging](#logging)
+9. [Advanced Configuration Options](#advanced-configuration-options)
 
 ## Intro
 
-The common events library is a module makes it easier to gather events from [Puppet API end points](https://puppet.com/docs/pe/2021.2/orchestrator_api_jobs_endpoint.html#orchestrator_api_jobs_endpoint) and send them to other platforms. Once the module is installed in your environment, you can create classes that wrap the API calls for you to gather the events you need.
+This module gathers events from the [Puppet Jobs API][1] and the [Activities API][2] using a script executed by a cron job. This data is provided to any available processor scripts during each run. The scripts are provided by any modules that want to make use of this data such as the [puppetlabs-splunk_hec][3] module. These processor scripts then handle the data they are given and forward it on to their respective platforms.
```

Comment:
We may want to make sure that this style of linking will work on the forge. Just a note to check once it's released.

Reply:
Yeah, we can keep an eye on this and fix if it's broken.